webserver/observe/metrics/src/main/java/module-info.java

```diff
@@ -26,6 +26,7 @@
          description = "WebServer Metrics support",
          in = HelidonFlavor.SE)
 module io.helidon.webserver.observe.metrics {
+    uses io.helidon.metrics.spi.MeterRegistryFormatterProvider;
```

Comment:
Please move `uses` to the end of the module info

Reply:
Done. Also changed to fully-qualified names, not imports.